airflow/migrations/versions/0041_3_0_0_remove_scheduler_lock_column.py

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+remove scheduler_lock column.
+
+Revision ID: 486ac7936b78
+Revises: 05234396c6fc
+Create Date: 2024-10-23 07:48:52.494396
+
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "486ac7936b78"
+down_revision = "05234396c6fc"
+branch_labels = None
+depends_on = None
+airflow_version = "3.0.0"
```

Comment:
If it's redundant and doesn't do anything why are we targeting 3.0 rather than next release?

Reply:
It can be backported and the version can be updated

Reply:
If we can do this in 2.10.x / 2.11 I think it's preferred but lets see what others think

Reply:
Since this can not be classified as a bug fix, I think doing it as part of 3.0 is ok. I prefer we don't backport it even, but it's not a strong opinion